scrapy/crawler.py

```diff
@@ -123,17 +161,43 @@ def _setup_crawler_logging(self, crawler):
             crawler.signals.connect(log_observer.stop, signals.engine_stopped)
 
     def stop(self):
+        """
+        Stops simultaneously all the crawling jobs taking place.
+
+        Returns a deferred that is fired when they all have ended.
+        """
         return defer.DeferredList([c.stop() for c in list(self.crawlers)])
 
     @defer.inlineCallbacks
     def join(self):
-        """Wait for all managed crawlers to complete"""
+        """
+        join()
```

Comment:
you can write a signature in the docs itself, see http://sphinx-doc.org/ext/autodoc.html#directive-automethod